articles/multifactor-authentication/duo/admin-guide.md

```diff
@@ -0,0 +1,136 @@
+# Duo for Administrators
+
+## Enabling Duo for MFA
```

Comment:
Shouldn't we warn the users about what would happen if they have guardian or other provider set up?